websecurityscanner/google/cloud/websecurityscanner_v1beta/gapic/web_security_scanner_client_config.py

```diff
@@ -65,7 +64,7 @@
                 },
                 "ListCrawledUrls": {
                     "timeout_millis": 60000,
-                    "retry_codes_name": "idempotent2",
+                    "retry_codes_name": "idempotent",
```

Comment:
@tseaver @crwilcox Do you know what "idempotent2" is?

Reply:
@busunkim96 It looks like *you* added it in #7992.

Reply:
`idempotent2` doesn't seem like a real retry code. Not sure how this ended up here, but this seems like the right change.

Reply:
Maybe it was part of the whole "don't retry on DEADLINE_EXCEEDED / oh, wait, never mind, do retry on DEADLINE_EXCEEDED." cycle we went through last week?

Reply:
I didn't see it anywhere else. Maybe something with the timing? woman_shrugging: